media/sdb/sysmain.xml

```diff
@@ -363,6 +390,24 @@
             <DATA NAME="SHIMVERSIONNT" DATA_DWORD="603" />
             <!-- TODO: Add more fixes! -->
         </LAYER>
+        <LAYER NAME="WIN10RTM">
+            <!-- ReactOS specific. Windows does not have this version lie -->
+            <SHIM_REF NAME="Win10RTMVersionLie" />
+            <DATA NAME="SHIMVERSIONNT" DATA_DWORD="A00" />
```

Comment:
Should be "1000"
'A00' on itself is not a valid number

Reply:
Fixed. Thanks.
Now that compilation warnings does no longer appear for me. And modes still work correctly (same as before).